Makefile

```diff
@@ -86,5 +86,10 @@ ko: export KO_DOCKER_REPO=us-central1-docker.pkg.dev/pantheon-sandbox/pantheon-s
 ko:
 	ko apply -f k8s/daemonset.yaml
 
+# requires protoc with go support
+# for macOS install with: brew install protoc-gen-go
```

Comment:
@jburianek can we drop a link to where we get the proto file from as well ?

We maybe want to update the proto on build in CI or something 

Should we generate the Go generated code every time in CI ? or we ok just generating and committing for now

Reply:
I am +1 on making CI pull the proto and gen the code on build will ensure we get a break if the proto changes, or if the project moves or w/e.

Reply:
@spheromak I already did 😄
https://github.com/pantheon-systems/cos-update-operator/pull/7/commits/3822ade9a68d21f90c98c272fb0540e9841527ac#diff-726e9219bfa9014ffcd7e8bb36d3fbc1R5

Reply:
And I'll see about generating the go protobuf code at build time in CI.

Reply:
Done, the circle build now downloads the proto file and generates go code from it as part of the build.